__tests__/sbom/main-test-app/verify-sbom.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+cd target
+
+required_patterns=(
+    '"pkg:maven/org.json/json@20211205"'
+    '"main-test-app"'
+    '"svm"'
+    '"nativeimage"'
+)
+
+for pattern in "${required_patterns[@]}"; do
+    echo "Checking for $pattern"
+    if ! grep -q "$pattern" sbom.sbom.json; then
```

Comment:
... end then:
```suggestion
    if ! grep -q "{pattern" "$script_dir"/target/sbom.sbom.json; then
```

Not sure how this can be done in a cmd file though.